### PR TITLE
API: CMS

### PR DESCRIPTION
**Pull request by rubenhorn**
Preview new API [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/CollActionteam/collaction_backend/doc/gh-66/cms-api/docs/api2.yaml).

closes #67

### CODE REVIEW

File: docs/api2.yaml [hunk 1]
```diff
@@ -479,16 +461,42 @@ paths:
...
+                      email: Not a valid email address
+        "403":
+          $ref: "#/components/responses/UnsupportedClientVersion"
+  /cms/crowdactions:
+    parameters:
+      - $ref: "#/components/parameters/ApiVersionParameter"
+    get:
+      tags:
+        - CMS
+      summary: List created crowdactions
```
Comment (Xazin) [inline docs/api2.yaml]:
```suggestion
      summary: List crowdactions
```

File: docs/api2.yaml [hunk 2]
```diff
@@ -498,9 +506,95 @@ paths:
...
+                    items:
+                      $ref: "#/components/schemas/CMScrowdactionListItem"
+        "403":
+          $ref: "#/components/responses/UnsupportedClientVersion"
+      security:
+        - CMSAuthorizer: []
+    post:
+      tags:
+        - CMS
+      summary: Created new crowdaction draft
```
Comment (Xazin) [inline docs/api2.yaml]:
```suggestion
      summary: Create a draft crowdaction
```

File: docs/api2.yaml [hunk 3]
```diff
@@ -510,11 +604,200 @@ paths:
...
-                      email: Not a valid email address
-        '403':
-          $ref: '#/components/responses/UnsupportedClientVersion'
+                      password: crowdaction does not exist
+      security:
+        - CMSAuthorizer: []
+    put:
+      tags:
+        - CMS
+      summary: Get details of a specific crowdaction
```
Comment (Xazin) [inline docs/api2.yaml]:
```suggestion
      summary: Update the details of a specific crowdaction
```

File: docs/api2.yaml [hunk 4]
```diff
@@ -498,9 +506,95 @@ paths:
...
+                          example: sustainability#food#88615462-2789-4159-8659-2ecfd33ef305
+        "403":
+          $ref: "#/components/responses/UnsupportedClientVersion"
+      security:
+        - CMSAuthorizer: []
+  "/cms/crowdactions/{crowdactionID}":
+    get:
+      tags:
+        - CMS
+      summary: Get details of a specific crowdaction
```
Comment (Xazin) [inline docs/api2.yaml]:
```suggestion
      summary: Get the details of a specific crowdaction
```

**Comment by Xazin** [comment]
How do you want to go about inviting and revoking admin access?

Ideally, we should have 3 groups:
- user
- admin
- super admin

Super Admin can invite and revoke admin access (Removes their ACP rights, but they're still users)

By that definition we should have some kind of endpoint to manage admins:
`/cms/administrators`:
- POST: Invites administrator by email
- DELETE: Revokes administrator rights (If not super admin)
- GET: Gets a list of all administrators (email, display name, phone number, role)
Opinion?

We also need to get their role by authentication

**Comment by rubenhorn** [comment]
closes #66

**Review by Xazin — APPROVED** [review]

**Pull request merged**